fintual-oss/paraffin-front
# Styles and mixins

### Context

With this update we could use Sass style as:

- Mixins
- Variables
- Fonts
- Breakpoints

Example to use font:
@import "font_faces.scss"
.some_class {
  font-family: 'Roboto', sans-serif;
}

We could use "Roboto" because is included in "font_faces.scss".

### Changelog

- The globals.scss file was updated importing the necessary files to work with: responsiveness, font faces, and variables.
- Was added scss files as: font_faces, variables, and folder mixins with files: breakpoints, and responsiveness
- The files: LearningUnitInformation.module.scss, and ResourceScroller.module.scss was updated, changing the breakpoint.

### QA

- Launch the front-end app with "yarn dev" in console
- Check the url "http://localhost:3001/curriculums/1" and inspect the font-family, this must to be "Roboto" in titles and text.
